src/assets/src/components/queueManager.tsx

```diff
@@ -243,17 +214,19 @@ const MeetingInfoDialog = (props: MeetingInfoDialogProps) => {
             </>
         );
 
-    const meetingType = props.meeting?.backend_type
-    const metadataInfo = meetingType
-        && (
-            VideoBackendNames.includes(meetingType)
-                ? (
-                    <HostVideoMeetingInfo
-                        backend={getBackendByName(meetingType, props.backends)}
-                        metadata={props.meeting!.backend_metadata!}
-                    />
-                ) : <div><p>This meeting will be <strong>In Person</strong>.</p></div>
-        );
+    const meetingType = props.meeting?.backend_type;
+    let metadataInfo;
+    if (props.meeting && meetingType && props.meeting.backend_metadata) {
+        const meetingBackend = getBackendByName(meetingType, props.backends);
+        metadataInfo = VideoBackendNames.includes(meetingType)
+            ? (
+                <>
+                <p>This meeting will be via <strong>{meetingBackend.friendly_name}</strong>.</p>
+                <DialInContent metadata={props.meeting.backend_metadata} backend={meetingBackend} />
+                </>
+            )
+            : <div><p>This meeting will be <strong>In Person</strong>.</p></div>;
+    }
```

Comment:
Not really related to your changes, but when I reread this whole block I wonder if it would make more sense to exit early if `!meeting` (thus assuring to TS that `meeting` is not undefined), rather than continuing along with a bunch of conditional checks and rendering that are ultimately pointless. Again, not for this PR, just something to think about in the future.

Reply:
Nice idea. I'll keep that in mind for the future, but yeah, that'd require modifying this whole component.